gradle/plugins/common/src/main/kotlin/junitbuild.java-library-conventions.gradle.kts

```diff
@@ -18,11 +18,17 @@ plugins {
 
 rewrite {
 	activeRecipe("org.openrewrite.java.migrate.UpgradeToJava17")
+	activeRecipe("org.openrewrite.staticanalysis.RemoveUnusedPrivateMethods")
+	activeRecipe("org.openrewrite.staticanalysis.MissingOverrideAnnotation")
+	failOnDryRunResults = true
 }
 
 dependencies {
-	rewrite(platform("org.openrewrite.recipe:rewrite-recipe-bom:latest.release"))
+	// https://github.com/openrewrite/rewrite/issues/5677
+	// rewrite(platform("org.openrewrite.recipe:rewrite-recipe-bom:latest.release"))
+	rewrite(platform("org.openrewrite.recipe:rewrite-recipe-bom:3.10.0"))
```

Comment:
This still fails when I run it locally: https://ge.junit.org/s/j7svvwjefwdrs/failures#1

Reply:
Next scheduled .minor release would be ~next Wednesday; not clear yet if we'd do a .patch release before then.

Great to see adoption here; also note that for one-off apply of recipes it's not necessary to adapt the builds just yet.